src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
+$card-size: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(1, $card-size);
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
You don't need a margin-bottom, because the distance between cards must be only 48px.

Reply:
Agree, remove it